app/javascript/lib/table-item-anchors.js

```diff
@@ -1,20 +0,0 @@
-document.addEventListener("DOMContentLoaded", function () {
-  [...document.querySelectorAll("a#back-button")].forEach(function (btn) {
-    btn.addEventListener("click", function (evt) {
-      evt.preventDefault();
-      history.back();
-    });
-    return btn;
-  });
-  [...document.querySelectorAll("table[data-anchor-rows] tr[id] a:not([href^='#'])")].forEach(
```

Comment:
I think we can remove the `data-anchor-rows` attributes from the various tables now.

Reply:
Good point, will do